opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -152,8 +152,8 @@ enum SeverityNumber {
 
 // Masks for LogRecord.flags field.
 enum LogRecordFlags {
-  LOG_RECORD_FLAG_UNSPECIFIED = 0;
-  LOG_RECORD_FLAG_TRACE_FLAGS_MASK = 0x000000FF;
+  LOG_RECORD_FLAGS_UNSPECIFIED = 0;
+  LOG_RECORD_FLAGS_TRACE_FLAGS_MASK = 0x000000FF;
 }
```

Comment:
I would rather remove them, since this repo is for the protocol definition and not for the "helpers" around the protocol.

Reply:
These help to implement the protocol correctly. Without these enums we have to explain the flags using English prose, which is more fragile and error prone. 

I understand the desire to minimize the contract surface but I believe to help increase OTLP popularity we need to take some pains ourselves in order to reduce the pains for others who adopt OTLP. See my comment about the proposed approach here: https://github.com/open-telemetry/opentelemetry-proto/issues/400#issuecomment-1161848977

Reply:
I disagree since a standard like w3c does not define "helpers" for trace flags for example. Also it is very bad design right now that helpers related to trace flags are defined in "logs" package.

I strongly suggest to remove for the moment, and determine if you are right or not about the "wrong" implementation, once we first hear about that we can have a small design for the helpers: where do they live, what are the guarantees since in this repo we mention that we guarantee wire not generated code (then how do you use this?), etc.

Reply:
>I disagree since a standard like w3c does not define "helpers" for trace flags for example.

I am not sure the analogy with w3c standard that you bring helps. The entire w3c trace context spec is just English prose, so there is no mechanism available for formal, compiler-enforced enum declarations there. I think the fact that w3c standard doesn't define helpers is not an argument that helps us to make a decision in our case, where such Protobuf compiler-enforced mechanism is luckily available.

>Also it is very bad design right now that helpers related to trace flags are defined in "logs" package.

I think there is some confusion about what this enum is. This helper is not trace flags definitions. This helper enum is the definition of `LogRecord.flags` field. Yes, 8 bits of the `flags` field will contain the value of w3c trace flags, but note how we don't define what is contained in those 8 bits here - the fact that only 1 bit is actually used here is irrelevant here, is not OTLP's business and is intentionally omitted. There may be other LogRecord flag definitions added here in the future, totally unrelated to the w3c trace flags. So, this is really `LogRecord.flags` field definition. Can you clarify why you think it is a bad design?

To be clear, I am not strongly opposed to removing the helpers. I do think the helpers are useful and have a slight preference to keep them, but if you see strong arguments in favour of removing we can remove them.

Reply:
I'm with @tigrannajaryan.   Soft-yes to include them, mostly because I don't see strong arguments for removing them.